SimulationConditions/inc/SimBookkeeperMaker.hh

```diff
@@ -0,0 +1,43 @@
+#ifndef SimulationConditions_SimBookkeeperMaker_hh
+#define SimulationConditions_SimBookkeeperMaker_hh
+
+//
+// Makes the SimBookkeeper ProditionsEntitiy
+//
+
+#include "SimulationConditions/inc/SimBookkeeper.hh"
+#include "SimulationConfig/inc/SimBookkeeperConfig.hh"
+#include "DbTables/inc/SimEfficiencies.hh"
+
+namespace mu2e {
+
+  class SimBookkeeperMaker {
+  public:
+    SimBookkeeperMaker(SimBookkeeperConfig const& config):_config(config) {}
+
+    typename SimBookkeeper::ptr_t fromFcl() {
```

Comment:
I'd prefer to put these implementations in a cc file, just so that all proditions have the same file pattern

Reply:
Sorry, I saw this one after I committed. I'm splitting the implementations into a cc file and am trying to resolve some compilation errors. It should be done soon.

Reply:
This is now done :)